doc/eks.md

```diff
@@ -0,0 +1,128 @@
+# Creating a Supercluster Compatible EKS Cluster
+
+This document explains how to use the `eks-init.py` script in the `scripts`
+directory to build a supercluster compatible EKS cluster on AWS.
+
+## Requirements
+
+The script requires that you have the following programs installed:
+* AWS CLI
+* eksctl
+* kubectl
+* helm
+* dig
+* ssh
+
+### Set up AWS CLI tools
+
+Before running the script, please follow
+[Amazon's guide for setting up tools to manage EKS clusters](https://docs.aws.amazon.com/eks/latest/userguide/setting-up.html).
+
+### AWS Quotas
+
+In order to run the theoretical max TPS test with the recommended parameters,
+you'll need a quota of at least 160 vCPUs for on-demand M instance types in the
```

Comment:
The instance type we recommend (`m5d.4xlarge`) has 16 vCPUs, and since we need to spin up 10 of them the AWS account needs to have a quota of at least 160. It's true that stellar-core doesn't use all of those vCPUs, but AWS bundles a bunch of other stuff we want with the larger instance (higher bandwidth, more IOPS, etc). If you're curious, there's a breakdown of all the stuff that gets bundled with each instance type here: https://docs.aws.amazon.com/ec2/latest/instancetypes/gp.html

Reply:
Makes sense, thanks!